src/shared/ui/modal/modal.module.scss

```diff
@@ -0,0 +1,76 @@
+.modal {
+  max-width: 889px;
+  padding: 0 32px 32px 32px;
+
+  line-height: 24px;
+
+  border: none;
+  border-radius: 12px;
+
+  &::backdrop {
+    background-color: hsl(0deg 0% 0% / 50%);
+  }
+
+  @include media-tablet {
+    padding: 0 16px 16px 16px;
+  }
+
+  @include media-tablet-large {
+    max-width: revert;
+  }
+}
+
+.modal-content {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.modal-header {
+  position: sticky;
+  top: 0;
+
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  padding: 26px 0 24px 0;
+
+  background-color: $color-white;
+
+  &.no-title {
+    justify-content: flex-end;
+  }
+
+  @include media-tablet {
+    padding: 12px 0 12px 0;
+  }
+}
+
+.modal-title {
```

Comment:
If we use our shared components (Subtitle, Paragraph) some styles can be removed

Reply:
Let's change  here `<h3> to `Subtitle` too